src/main/java/stsjorbsmod/patches/WitheringPatch.java

```diff
@@ -0,0 +1,48 @@
+package stsjorbsmod.patches;
+
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.IntangiblePlayerPower;
+import javassist.CtBehavior;
+import stsjorbsmod.cards.cull.Withering;
+import stsjorbsmod.powers.WitherPower;
+
+public class WitheringPatch {
+    @SpirePatch(clz = IntangiblePlayerPower.class, method = "atDamageFinalReceive")
+    public static class ReduceIntangibleDamageReductionPatch
+    {
+        @SpirePrefixPatch
+        public static SpireReturn patch(IntangiblePlayerPower __instance, float dmg, DamageInfo.DamageType type) {
+            if(AbstractDungeon.player.hasPower(WitherPower.POWER_ID) && dmg > 1)
```

Comment:
Probably better to test against `__instance.owner` rather than `AbstractDungeon.player` here (some particularly wild other mods add multiple "player" entities)